### PR TITLE
BXMSDOC-3284 Managing and monitoring Process Server (For PAM 7.1)

### PR DESCRIPTION
**Pull request by gadeyredhat**
Please review suggested changes to the **Managing and monitoring Process Server (For PAM 7.1)** document.
[Rendered Output](http://file.pnq.redhat.com/~gadey/BXMSDOC-3284/#migration-configure-kie-server-proc)
[Jira ID](https://issues.jboss.org/browse/BXMSDOC-2934)

### CODE REVIEW

**Review by MarianMacik — APPROVED** [review]
Looks cool @gadeyredhat. Thanks for the changes.

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc [hunk 1]
```diff
@@ -15,17 +15,25 @@ Navigate to the {PRODUCT} {PRODUCT_VERSION} `bin` directory and start the new {K
...
 -Dorg.kie.server.id=myserver <2>
--Dorg.kie.server.controller=http://localhost:8080/{URL_COMPONENT_CENTRAL}/rest/controller <3>
--Dorg.kie.server.location=http://localhost:8080/kie-server/services/rest/server <4>
--Dorg.kie.server.persistence.dialect=org.hibernate.dialect.PostgreSQLDialect <5>
--Dorg.kie.server.persistence.ds=java:jboss/datasources/psjbpmDS <6>
+-Dorg.kie.server.user=username <3>
+-Dorg.kie.server.pwd=password <4>
+-Dorg.kie.server.controller=http://localhost:8080/{URL_COMPONENT_CENTRAL}/rest/controller <5>
+-Dorg.kie.server.controller.user=process_server_username <6>
+-Dorg.kie.server.controller.pwd=process_server_password <7>
```
Comment (MarianMacik) [inline doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: This should be something like controller_username and password. And process_server_username/password should be for org.kie.server.user/pwd parameter.

**Review by MarianMacik — CHANGES_REQUESTED** [review]
After the second look I realized that the values can be improved. See the comment.

File: doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc (hunk 1 above)
Comment (gadeyredhat) [reply doc-content/enterprise-only/migration/migration-configure-kie-server-proc.adoc]: Changed as suggested.

**Pull request merged**